### DISCUSSION TITLE
JamesDSP to manage Pipewire - your take on it.

### DISCUSSION THREAD

**Discussion by dacomboman** [SpiralLinux/SpiralLinux-project#357] [General]
Flatpak has its issues, so trying out .deb which seems to be doing pretty good :
https://github.com/Audio4Linux/JDSP4Linux/releases/tag/2.6.1

**Comment by geckolinux** [comment]
Never heard of it, but looks interesting, thanks!

**Reply by dacomboman** [reply]
You are welcome, and thank you for this nice Cinnamon ed.! 

Btw, like i said, the .deb file works best (at least for me).

**Reply by dacomboman** [reply]
After a few days of various issues, decided to remove its .deb app. for now. 

Have gone to testing EasyEffects in the meantime.

**Comment by dacomboman** [comment]
Noticed 2.7 has been out for a little while.

Giving it another shot.

Hope to find configuration that suits me best and no glitches.
